EntityFramework/HomeWork3/Program.cs

```diff
@@ -0,0 +1,206 @@
+﻿using FoodOrderingSystem.Models;
+using FoodOrderingSystem.Repositories;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.IO;
+using System.Text;
+
+namespace FoodOrderingSystem
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.OutputEncoding = Encoding.UTF8;
+
+            var config = Initialization();
+
+            var categoryRepositoryDapper = new CategoryRepositoryDapper(config.GetConnectionString("DefaultConnection"));
+            var categoryRepositoryDapperContrib = new CategoryRepositoryDapperContrib(config.GetConnectionString("DefaultConnection"));
+
+            //сохранение сущности без вложенных объектов (Dapper)
```

Comment:
for now let it be 
but in the future use only (with common sense) english words
and try not tu use comments
if u need explanation - u can use separate method with meaningful name
or meaningful variable name